lua/ttt2/libraries/weaponrenderer.lua

```diff
@@ -58,7 +58,7 @@ local function GetBoneOrientation(wep, baseDataTable, dataTable, boneEntity, bon
 
         -- Technically, if there exists an element with the same name as a bone
         -- you can get in an infinite loop. Let's just hope nobody's that stupid.
-        pos, ang = GetBoneOrientation(wep, baseDataTable, tbl, boneEntity)
+        pos, ang = GetBoneOrientation(wep, baseDataTable, tbl, boneEntity, isInWorld)
```

Comment:
you could store all looked up bone names and stop once you reached an earlier one to not get into an infinite loop.

Reply:
That's part of the original code from the SWEP construction Kit. Admittedly not much is left of that by now. Your suggestion is probably good, but I think out of scope for this PR. Also this is an issue that never happened to me